ibis/mapd/client.py

```diff
@@ -766,6 +813,23 @@ def get_schema(self, table_name, database=None):
             for col in self.con.get_table_details(table_name)
         ])
 
+    def sql(self, query):
```

Comment:
Is there a way to reuse the parent's method here, or is does this need to be overridden?